File: Easy/9. Palindrome Number/leet.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if(x < 0):
            return False
        x = str(x)
        i = len(x) - 1
        tmp = str()
        while i >= 0:
            tmp += x[i]
            i -= 1
        
        return tmp == x

File: Easy/9. Palindrome Number/leet.php
<?php 
class Solution {

/**
 * @param Integer $x
 * @return Boolean
 */
function isPalindrome($x) {
    $tmp = str_split($x);
    $tmp2 = array();
    for($i = (count($tmp)-1) ; $i >= 0 ; --$i)
        $tmp2[] = $tmp[$i];
    return $tmp === $tmp2;
}
}